<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistributionDialog</class>
 <widget class="QDialog" name="DistributionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select Distribution</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>14</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="Layout1" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="Layout11">
       <item>
        <widget class="QTabWidget" name="tabs">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabUniform">
          <attribute name="title">
           <string>Uniform</string>
          </attribute>
          <layout class="QVBoxLayout" name="P1Layout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="P1Layout1">
             <item>
              <widget class="QLabel" name="lblUniformMin">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Min</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sldUniformMin">
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>10</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtUniformMin">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>10</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="P1Layout2">
             <item>
              <widget class="QLabel" name="lblUniformMax">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Max</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sldUniformMax">
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtUniformMax">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>500</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="P1Spacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabPareto">
          <attribute name="title">
           <string>Pareto</string>
          </attribute>
          <layout class="QVBoxLayout" name="P2Layout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>12</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="P2Layout1">
             <item>
              <widget class="QLabel" name="lblParetoMean">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Mean</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sldParetoMean">
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtParetoMean">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>500</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="P2Layout2">
             <item>
              <widget class="QLabel" name="lblParetoShape">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Shape</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sldParetoShape">
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>1500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtParetoShape">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>1.5</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="P2Layout3">
             <item>
              <widget class="QLabel" name="lblParetoBound">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Bound</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sldParetoBound">
               <property name="maximum">
                <number>10000</number>
               </property>
               <property name="value">
                <number>1450</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtParetoBound">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>1450</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="P2Spacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabExponential">
          <attribute name="title">
           <string>Exponential</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>291</width>
             <height>131</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="P3Layout">
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>12</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="P3Layout1">
              <item>
               <widget class="QLabel" name="lblExponentialMean">
                <property name="minimumSize">
                 <size>
                  <width>45</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Mean</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sldExponentialMean">
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>500</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtExponentialMean">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>500</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="P3Layout2">
              <item>
               <widget class="QLabel" name="lblExponentialBound">
                <property name="minimumSize">
                 <size>
                  <width>45</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Bound</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="sldExponentialBound">
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>1450</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtExponentialBound">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>1450</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="P3Spacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtDistribution">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Layout12" stretch="0,0">
       <item>
        <widget class="PdfWidget" name="pdfWidget" native="true"/>
       </item>
       <item>
        <widget class="CdfWidget" name="cdfWidget" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CdfWidget</class>
   <extends>QWidget</extends>
   <header>widget/cdfwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PdfWidget</class>
   <extends>QWidget</extends>
   <header>widget/pdfwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DistributionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DistributionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
